routes/web.php

```diff
@@ -13,12 +13,15 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
-});
 
-Route::get('/dashboard', function () {
+
+Route::get('/', function () {
     return view('admin.dashboard');
 })->middleware(['auth'])->name('dashboard');
 
+Route::get('profile/{user}',[UserController::class,'profile'])->name('profile');
+Route::get('user/list',[UserController::class,'index'])->name('user-list');
+Route::delete('user/delete/{user}',[UserController::class,'destroy'])->name('user-delete');
```

Comment:
The route should be `users/{user}` with the DELETE method.

Reply:
done